UserGuide.md

```diff
@@ -16,6 +16,17 @@ First, add the function definitions here:
 
```

Comment:
BTW, this is a file I'm keeping for reference for anyone who wants to start using Penrose…